sdk/eventhub/Azure.Messaging.EventHubs/tests/Infrastructure/EventDataExtensionTests.cs

```diff
@@ -28,6 +29,180 @@ public void IsEquivalentToDetectsDifferentBodies()
             Assert.That(firstEvent.IsEquivalentTo(secondEvent), Is.False);
         }
 
+        /// <summary>
+        ///   Verifies functionality of the <see cref="EventDataExtensions.IsEquivalentTo" /> test
+        ///   helper.
+        /// </summary>
+        ///
+        [Test]
+        public void IsEquivalentToDetectsDifferentDifferentOffsets()
```

Comment:
super nit: I think we should stick to a naming convention: offset or offsets (and we should follow the same convention for other members). There's some inconsistency in this file.

Reply:
One more thing: the test name has a repeated word (IsEquivalentToDetectsDifferent**Different**Offsets). Other tests in the same file have the same problem.

Reply:
Offset == 1
Offsets == more than 1.

In this particular case, we are comparing two different offsets to each other.

I *think* I'm using that consistently, but will review.

Reply:
Nope, I wasn't consistent.   Fixed.